### ISSUE TITLE
Getting a warning, normals not found when running ns-train dn-splatter

### ISSUE THREAD

**Issue by rowellz** [maturk/dn-splatter#51]
Hello! Thank you for this awesome project, I am trying to get this running on a monocular estimation dataset. Specifically the sauna one found [here](https://zenodo.org/records/10230733). When I get to `ns-train dn-splatter [OPTIONS]` I get this error `WARNING: You have enabled normal supervision with monocular normals but none were found.`

For context I am running this in a docker container based off this Dockerfile: 

```
FROM nerfstudio/nerfstudio:latest

USER root

RUN git clone https://github.com/maturk/dn-splatter

RUN pip install --upgrade pip

RUN cd dn-splatter && \
    pip install -e . 

WORKDIR /workspace
```

I shell into the container and run these commands in this order: 

`ns-process-data images --data /workspace/shared/sauna/ --output-dir /workspace/shared/sauna/`

`python3 /workspace/dn-splatter/dn_splatter/scripts/normals_from_pretrain.py --data-dir /workspace/shared/sauna/ --model-type dsine`

Which outputs: `Completed generating mono-normals!`

I then run `python3 /workspace/dn-splatter/dn_splatter/scripts/align_depth.py --data /workspace/shared/sauna/`

Which outputs: 

Finally I run: `ns-train dn-splatter --pipeline.model.use-normal-loss True --pipeline.model.normal-supervision mono coolermap --data /workspace/shared/sauna/ --normals-from pretrained --normal-format opencv`

Which outputs:

`WARNING: You have enabled normal supervision with monocular normals but none were found.`

I suspect the order in which I am running the commands might be incorrect, or perhaps I am missing a command entirely? Regardless, I am a bit surprised by the warning message, the normal maps generated in the /normals_from_pretrain/ dir look good.

**Comment by Eki87** [comment]
I got the same problem and make it by adding 
`--load-normals True`

**Comment by maturk** [comment]
Correct. Each dataparser has additional flags, like --load-depths or --load-normals which loads additonal data into memory. These are configurable options, since it is possible that some datasets (or user preference) do not have certain data. Memory consuption is also quite high since to run dn-splatter you need both depth and normal data along with rgb data.
